sdk/keyvault/keyvault-certificates/src/transformations.ts

```diff
@@ -390,3 +389,28 @@ export function getPropertiesFromCertificateBundle(
 
   return abstractProperties;
 }
+
+export function mapPagedAsyncIterable<T, U>(
```

Comment:
This probably could be code-gen'd now that I think about, part of the pagingHelpers maybe?

Reply:
Ah yeah, good idea

Reply:
keyvault-common could be a good option too if the helper gets too Key Vault specific (which might happen if we make the changes for maxPageSize)